test/test-network/conf/25-ipv6-retrans-time-toobig.network

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+[Match]
+Name=test25
+
+[Network]
+IPv6RetransmissionTimeSec=999999999999999999
```

Comment:
Not a matter, but `infinity` can be used for testing such.

Reply:
Testing for `infinity` is different to a value that is `too big`.  I've added a test for `infinity` too. When `infinity` is selected, it is ignored.